File: AtlasModel/Enterprise/TTPermissionParameterDef.cs
using System;

namespace AtlasModel.Enterprise
{
    public partial class TTPermissionParameterDef
    {
        public Guid ParameterDefId { get; set; }
        public Guid PermissionDefId { get; set; }
        public Guid DataTypeId { get; set; }
        public string Name { get; set; }
        public byte OrderNo { get; set; }
        public short CheckOutStatus { get; set; }
    }
}

File: AtlasWeb/Core/wwwroot/app/NebulaClient/StorageManager/DefinitionManagement/TTPermissionParameterDef.ts
import { Guid } from "../../Mscorlib/Guid";
import { TTPermissionDef } from "./TTPermissionDef";
import { TTDataType } from "../../DataDictionary/TTDataType";

export class TTPermissionParameterDef {
    public ParameterDefID: Guid;
    public PermissionDef: TTPermissionDef;
    public PermissionDefID: Guid;
    public DataType: TTDataType;
    public DataTypeID: Guid;
    public Name: string;
    public OrderNo: number;
    public ToString(): string {
        return null;
    }
}
